fail2ban/tests/files/logs/guacamole

```diff
@@ -1,12 +1,2 @@
-# failJSON: { "match": false }
-apr 15, 2013 8:34:08 PM org.slf4j.impl.JCLLoggerAdapter warn
-# failJSON: { "time": "2013-04-15T20:34:08", "match": true , "host": "192.0.2.0" }
-WARNING: Authentication attempt from 192.0.2.0 for user "null" failed.
-# failJSON: { "match": false }
-apr 16, 2013 8:32:13 AM org.slf4j.impl.JCLLoggerAdapter warn
-# failJSON: { "time": "2013-04-16T08:32:13", "match": true , "host": "192.0.2.0" }
-WARNING: Authentication attempt from 192.0.2.0 for user "null" failed.
-# failJSON: { "match": false }
-apr 16, 2013 8:32:28 AM org.slf4j.impl.JCLLoggerAdapter warn
-# failJSON: { "time": "2013-04-16T08:32:28", "match": true , "host": "192.0.2.0" }
-WARNING: Authentication attempt from 192.0.2.0 for user "pippo" failed.
+# failJSON: { "time": "12:57:32", "match": true , "host": "182.23.72.36" }
+12:57:32.907 [http-nio-8080-exec-10] WARN  o.a.g.r.auth.AuthenticationService - Authentication attempt from 182.23.72.36 for user "guacadmin" failed.
```

Comment:
The only thing I'm not sure about is here, `time` argument.
I don't know whether or not it will match like this...

Reply:
So, it fails.
Not sure what's the best way to solve this, could be to add the date to the `datepattern` (something like `^%%Y-%%m-%%d %%H:%%M:%%S.%%f`), but then we would go away from the default Guacamole log pattern.
I'll need your advice @sebres. Thx 👍

Reply:
1. Fail2ban always assumes today if it finds a time without date
2. As you can see here, fail2ban test cases are working in "Aug 14 12:00:00 CEST 2005": https://github.com/fail2ban/fail2ban/blob/ac41b8f3be61fedf6cea724d4d1775c07d2045da/fail2ban/tests/utils.py#L359
so you could simply provide full date-time string with `"time": "2005-08-14T12:57:32"`, but if I'm not wrong we had some bug in test-cases that can consider this date as `2005-08-13T12:57:32` (as previous day).

I have to check that.

Reply:
Thank you for your detailed answer 👍
I just tested, to be sure / to help, and you're right, it fails as you suspected :
```
AssertionError: guacamole: UTC Time  mismatch 1123930652.0 (2005-08-13T10:57:32) != 1124017052.0 (2005-08-14T10:57:32)  (diff -86400.000 seconds)
```

Reply:
> if I'm not wrong we had some bug in test-cases that can consider this date as `2005-08-13`...

OK, not a bug but feature - if parsed time is in the feature `"12:57:32" > now ("12:00:00")` assume yesterday as date, so I guess it will be `2005-08-13T12:57:32` not the 14th.